py/server/deephaven/table.py

```diff
@@ -2822,21 +3198,29 @@ def filter(self, filters: Union[str, Filter, Sequence[str], Sequence[Filter]]) -
         """
         filters = to_sequence(filters)
         if isinstance(filters[0], str):
-            filters = Filter.from_(filters)
+            filters = Filter.from_(filters)  # type: ignore[arg-type]
```

Comment:
would
``` filters = to_sequence(Filter.from_(filters))``` avoid this ignore?

Reply:
No. mypy isn't smart enough to infer the `filters` is a sequence of string.